swap/peer.go

```diff
@@ -197,6 +197,16 @@ func (p *Peer) sendCheque() error {
 		return fmt.Errorf("error while creating cheque: %v", err)
 	}
 
+	chequebookBalance, err := p.swap.AvailableBalance()
```

Comment:
Could you just call this availableBalance. ChequebookBalance is not correct, as availableBalance is less than the chequebookBalance (see the defenition of availableBalance)

Reply:
you're right, fixed